src/js/helpers/shareMedia.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+module.exports = function (client, stanzas) {
+    var types = stanzas.utils;
+    let jxt = require('jxt').createRegistry();
```

Comment:
Again, you have JXT instance in `stanzas` variable and utils reference in `types`, no need to instantiate new JXT object, and new stanzas definitions should extend `stanzas` object

Reply:
done